daemon/emer-network-send-provider.c

```diff
@@ -191,15 +191,19 @@ reset_network_send_data (EmerNetworkSendProvider *self)
                   error->message);
       g_error_free (error);
     }
+
+  priv->send_number = 0;
+  priv->data_cached = TRUE;
 }
 
-static gboolean
+static void
 read_network_send_data (EmerNetworkSendProvider *self)
 {
   EmerNetworkSendProviderPrivate *priv =
     emer_network_send_provider_get_instance_private (self);
+
   if (priv->data_cached)
```

Comment:
Cosimo points out that upstream already officially mentions that `g_object-new` zeroes out members of the private struct at https://developer.gnome.org/gobject/stable/chapter-gobject.html#gobject-instantiation.